tinygrad/tensor.py

```diff
@@ -511,7 +511,7 @@ def conv_transpose2d(self, weight:Tensor, bias:Optional[Tensor]=None, groups=1,
     padding = flatten((((k-1)*d-p,(k-1)*d-p+op) for k,d,p,op in reversed(list(zip(HW, make_pair(dilation, len(HW)), make_pair(padding, len(HW)), make_pair(output_padding, len(HW)))))))
     return x.conv2d(w.reshape(w.shape[0]*w.shape[1],*w.shape[2:]), groups=groups, bias=bias, dilation=dilation, padding=padding)
 
-  wino = int(getenv("WINO", "0"))
+  wino, igemm = int(getenv("WINO", "0")), int(getenv("IGEMM", "0"))
```

Comment:
can be `wino, igemm = getenv("WINO"), getenv("IGEMM")` because `getenv` gets the type from default value (int 0)

also probably need a test that covers this flag

Reply:
Good call on the test - will add it.